Fixes import bug with tippy shift-away.css

SomethingNew71 opened the pull request:
Hey John,

After pulling the update into my external project I started receiving an import failure on the `~` reference in the `main.scss` file. It appears my project was attempting to import tippy in your scss. 

I was looking at tippy's docs:
https://atomiks.github.io/tippyjs/v5/animations/#material-filling-effect

I think when I updated tippy I did not include the plugin reference correctly nor did I include the css as I should have. I have built this locally and replaced the npm installed version in my project with this version and it appears to be working correctly now.

johndatserakis commented:
Hey Cole - sorry about that. 

Hmm I see, I actually removed the `animateFill` because when I read the new tippy docs a bit it wanted us to also import `'tippy.js/dist/backdrop.css'` it seemed, which caused a really ugly background in my testing. So I kinda felt like I had been using it wrong this whole time. (I might be wrong here.)

I deduced that it wasn't really `animateFill` that was giving us the animation we wanted - it's really all because of the `tippy.js/animations/shift-away.css` import that is now required to be explicitly imported in newer versions of tippy. So I figured the `animateFill` is no longer needed. Additionally, the animation in the demo (without the `animateFill` plugin) is looking fine so it kinda backs this logic.

I'm totally down with importing `tippy.js/animations/shift-away.css` directly in `src/components/DesktopMenuItemLink.vue` like you have it though, to get rid of the other issue you mentioned and remove any ambiguity on that import.

Do you mind taking a look at the result of removing the `animateFill` entirely in your PR and seeing if it is something we can remove from the project?

Also I did update the PR to point at `master` for now - sorry for the confusion - at some point in the future I'll adjust that repo settings and make the main branch `develop`.

Finally, I see that there's an urgent issue in the code I pushed last night, so I'm going to fix that really quick. It'll be pushed at `2.1.1` Please merge in those changes and I'll help with any merge conflicts.

Thank you

SomethingNew71 commented:
No worries I made a mistake on that as well. I rebased in master so that is all gravy now. Based on your above comment I started testing it without `animateFill` used at all and things look to be working nicely for all the animations. Although the other options not being imported directly seemed to no longer function. I did see that for the newer version of `tippy.js` they say the following:

![Screen Shot 2020-07-08 at 9 11 34 AM](https://user-images.githubusercontent.com/4572799/86922746-13d0d980-c0fb-11ea-8ad8-f962a03a3a9f.png)

So I went ahead and imported the animations that you offer as options in the component. It looks like `fade` is no longer something tippy includes with their library so I also removed that from both the docs and the default configuration for the component.

johndatserakis commented:
That's great Cole thank you. I really appreciate you double-checking that. Merging this in now, taking a look at your other PR, and looking to get a release out tonight. Thank you.
